src/compas_viewer/components/renderer/shaders/model.vert

```diff
@@ -2,20 +2,20 @@
 
 attribute vec3 position;
 attribute vec3 color;
-attribute float alpha;
+// attribute float alpha;
 
 uniform mat4 projection;
 uniform mat4 viewworld;
 uniform mat4 transform;
 
 varying vec3 vertex_color;
-varying float vertex_alpha;
+// varying float vertex_alpha;
 varying vec3 ec_pos;
 
 void main()
 {
     vertex_color = color;
-    vertex_alpha = alpha;
+    // vertex_alpha = alpha;
```

Comment:
@tomvanmele This was the source of evil...

Reply:
Could you please merge this culprit?